src/skin/popup.html

```diff
@@ -80,5 +80,6 @@ <h1 id="report_title" class="i18n_report_title"></h1>
   <center><button id="donate" class="pbButton" style="display:block" target="blank"><span class="i18n_donate_to_eff"></span></button></center>
 </div>
 <div class='clear'></div>
+<div class='footer'><span id="version" class="i18n_version"></span></div>
```

Comment:
Hm, I didn't see your comment before I came back to work on this, and it turned out the missing link was including `float` for `version`. Does this most recent version look OK:

![screenshot from 2017-05-01 21-31-54](https://cloud.githubusercontent.com/assets/4513790/25604267/1152302a-2eb7-11e7-8362-80a191de4ffd.png)